challenge-1/picture-of-the-day/src/services/index.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-alert */
+import axios from 'axios';
+
+export async function getPicture(date) {
+  const apiKey = process.env.REACT_APP_API_KEY;
+  const searchUrl = `https://api.nasa.gov/planetary/apod?api_key=${apiKey}&date=${date}`;
+  try {
+    const response = await axios({
+      url: searchUrl,
+      method: 'GET',
+    });
+    return response;
+  } catch (error) {
+    const { code, msg } = error.response.data;
+    if (code === 'UNEXPECTED_ERROR') {
+      alert('There was an error, please try again.');
+      return;
+    }
+    console.log('error code ', code);
+    alert(msg);
```

Comment:
Please have into consideration that console logs and alerts are not the best way to retrieve data from the user. I would suggest you printing the message to the user another way that is more "friendly"

Reply:
should I implement a modal component?